content/post/071-ipfs-in-web-browsers.md

```diff
@@ -3,14 +3,15 @@ date: 2019-10-08
 url: 2019-10-08-ipfs-browsers-update
 title: IPFS Browser Update
 author: Dietrich Ayala
+header_image: 071-ipfs-in-web-browsers.png
 ---
 Decentralization can feel slow in providing user-centric approaches to the internet's biggest problems. Right now, software that is fully in the control of the user is often too technical, too fragile, and too time-consuming to be the default choice.
 
 But we're making headway. Today, we'd like to share some collaborations the IPFS project has had in the works for a while, which bring us a few steps closer to making unmediated access to information *just work*... by solving that "last mile" problem and *integrating IPFS directly into web browsers*.
 
 ## Stages
 
-![Stages of browser integration](https://file.globalupload.io/tF56ELOMoe.png)
+![Stages of browser integration](../img/071-ipfs-in-web-browsers/stages-of-browser-integration.png)
```

Comment:
This image isn't showing, and I can't find it in the repo.

Reply:
added!